pipelines/pipeline_katello_nightly_split.yml

```diff
@@ -0,0 +1,116 @@
+- hosts: localhost
+  vars:
+    forklift_name: pipeline-katello-nightly-split
+    forklift_boxes:
+      pipeline-app-nightly-centos7-split:
+        box: centos7
+        memory: 4096
+        ansible:
+          group: "{{ forklift_name }}"
+      pipeline-services-nightly-centos7-split:
+        box: centos7
+        memory: 4096
+        ansible:
+          group: "{{ forklift_name }}"
+      pipeline-proxy-nightly-centos7-split:
+        box: centos7
+        memory: 3072
+        ansible:
+          group: "{{ forklift_name }}"
+  roles:
+    - forklift
+
+- hosts:
+  - pipeline-app-nightly-centos7-split
+  - pipeline-proxy-nightly-centos7-split
+  - pipeline-services-nightly-centos7-split
+  become: yes
+  vars:
+    katello_repositories_version: nightly
+    foreman_repositories_version: nightly
+    katello_repositories_use_koji: true
+    foreman_installer_skip_installer: true
+    foreman_installer_additional_packages:
+      - foreman-installer-katello
+    foreman_installer_module_prs: 'katello/katello/209,katello/foreman_proxy_content/136'
+  roles:
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - disable_firewall
+    - haveged
+    - update_os_packages
+    - foreman_installer
+
+- hosts: pipeline-services-nightly-centos7-split
+  become: yes
+  vars:
+    foreman_installer_scenario: katello
+    foreman_installer_options_internal_use_only:
+      - "--disable-system-checks"
+      - "--no-enable-foreman"
+      - "--no-enable-foreman-cli"
+      - "--no-enable-foreman-plugin-tasks"
+      - "--no-enable-foreman-proxy"
+      - "--no-enable-foreman-proxy-content"
+      - "--no-enable-foreman-proxy-plugin-pulp"
```

Comment:
We need the three above this or else the services box won't have a proxy running on it (which is required).

Reply:
Good point, hadn't reached a point yet where I could actually use the service.

Reply:
It does mean we have a dependency cycle: smart proxy wants to register to foreman, foreman isn't provisioned yet but needs Candlepin in turn. Deploying a server with Candlepin means we have to fix the CRL in Pulp. A simple crl-updater script might be sufficient.